### PR TITLE
Ajax

### PR DESCRIPTION
**Pull request by mikeheft**
Additions: 

ajax call to admin landing page
User Story:

Concerns (code location):
admin stores index

Mentions to collaborators:
@amhursh @blsrofe @JunePaloma 

Notes(if applicable):

needs styling will work on that. It's only one ajax call but the concept is there

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -46,3 +46,16 @@ window.onclick = function(event) {
...
+  url: 'http://localhost:3000/api/v1/merchants',
+  headers: {'Authorization': 'Token token=5a9b5918e8fff9b0ca6f8c643422e8d1'},
+  }).done(function(data) {
+    console.log(data);
+    data.forEach(function(store) {
+      console.log(store);
+      $('#jq_ajax').append(
+      `<p>${store['name']}</p>
+      <p>${store['description']}</p>`
+  )})});
```
Comment (amhursh) [inline app/assets/javascripts/application.js:61]: Nice. I like the merchants data

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -14,6 +14,12 @@
 <% end %>
 </div>
 
+  <% if current_operator.platform_admin? %>
```
Comment (amhursh) [inline app/views/admin/dashboard/index.html.erb:17]: Yeah this should handle the operator differentiation.

**Review by amhursh — APPROVED** [review]
Looks good! Ready for merge

**Comment by mikeheft** [comment]
@amhursh added basic styling just now

**Pull request merged**